modules/oe_theme_content_entity_organisation/oe_theme_content_entity_organisation.module

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * Module file is used for theming the Organisation entity.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Url;
+use Drupal\Component\Utility\UrlHelper;
+use Drupal\file\FileInterface;
+use Drupal\media\MediaInterface;
+use Drupal\oe_theme\ValueObject\ImageValueObject;
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_content_entity_organisation_preprocess_oe_organisation(&$variables): void {
+  $entity = $variables['entity'];
+  // Prepare fields.
+  $fields_values = ['name' => 'name', 'oe_acronym' => 'acronym'];
+  foreach ($fields_values as $entity_field => $pattern_field) {
+    if (!$entity->get($entity_field)->isEmpty()) {
+      $variables[$pattern_field] = $entity->get($entity_field)->value;
+    }
+  }
+
+  // Extract the image if Media and File entities exist.
+  $media_image = $entity->get('oe_logo')->entity;
```

Comment:
We need to make sure that the entity is in the correct language here. Use \Drupal::service('entity.repository')->getTranslationFromContext().

Reply:
Fixed